src/happy/oauth2_capture_redirect.clj

```diff
@@ -38,9 +86,13 @@
         (.stop server)))))
 
 (defn request-code [config scopes optional]
```

Comment:
In contrast to `wait-for-redirect`, the `request-code` API and all its callers remain unbroken. My assumption is, that people usually would use the documented `auth!` and not those internal functions.